src/Tasks/Common/Resources/xlf/Strings.es.xlf

```diff
@@ -812,8 +812,8 @@ The following are names of parameters or literal values and should not be transl
       </trans-unit>
       <trans-unit id="SelfContainedOptionShouldBeUsedWithRuntime">
         <source>NETSDK1179: One of '--self-contained' or '--no-self-contained' options are required when '--runtime' is used.</source>
-        <target state="translated">NETSDK1179: Se requiere una de las opciones "--self-contained" o "--no-self-contained" cuando se usa "--runtime".</target>
-        <note>{StrBegin="NETSDK1179: "}</note>
+        <target state="new">NETSDK1179: One of '--self-contained' or '--no-self-contained' options are required when '--runtime' is used.</target>
```

Comment:
Shouldn't `--runtime` also be locked?

Reply:
Good catch. I also found a --output in the same file so fixed it as well.